unity/Holo/Assets/Scripts/CustomizeMrtk.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.Windows.Speech;
+using Microsoft.MixedReality.Toolkit.Core.Services;
+using Microsoft.MixedReality.Toolkit.Core.Definitions;
+using System.Reflection;
+
+/* TODO: Work-in-progress, doesn't work yet. I tried numerous ways to disable DictationSystem at runtime,

```

Comment:
Why you want to turn off DictationSystem? 
I suppose that you want to get rid of errors while having this solution on different platform than Windows 10?

I think switching to HoloToolkit MRTK should do most of the tricks as MRTK currently included is still in developing phase and couple things are missing.

Reply:
Testing on Windows 10 Pro, if I don't enable "Speech Recognition" (or something like that) system-wide, the Unity console is flooded with errors (one error each frame).

At the beginning there's one error:

```
UnityException: Speech recognition is not supported on this machine.
UnityEngine.Windows.Speech.KeywordRecognizer..ctor (System.String[] keywords, UnityEngine.Windows.Speech.ConfidenceLevel minimumConfidence) (at C:/buildslave/unity/build/Runtime/Export/Windows/Speech.cs:221)
Microsoft.MixedReality.Toolkit.Providers.WindowsVoiceInput.WindowsSpeechInputProvider.Enable () (at Assets/MixedRealityToolkit.Providers/WindowsVoiceInput/WindowsSpeechInputProvider.cs:69)
Microsoft.MixedReality.Toolkit.Core.Services.MixedRealityToolkit.EnableAllServices () (at Assets/MixedRealityToolkit/Services/MixedRealityToolkit.cs:945)
Microsoft.MixedReality.Toolkit.Core.Services.MixedRealityToolkit.OnEnable () (at Assets/MixedRealityToolkit/Services/MixedRealityToolkit.cs:565)
```

But then in every Update, there's an error:

```
NullReferenceException: Object reference not set to an instance of an object
Microsoft.MixedReality.Toolkit.Providers.WindowsVoiceInput.WindowsDictationInputProvider.Update () (at Assets/MixedRealityToolkit.Providers/WindowsVoiceInput/WindowsDictationInputProvider.cs:113)
Microsoft.MixedReality.Toolkit.Core.Services.MixedRealityToolkit.UpdateAllServices () (at Assets/MixedRealityToolkit/Services/MixedRealityToolkit.cs:966)
Microsoft.MixedReality.Toolkit.Core.Services.MixedRealityToolkit.Update () (at Assets/MixedRealityToolkit/Services/MixedRealityToolkit.cs:570)
```

The latter error makes it hard to test the application, as our actual errors would be lost in the flood of errors from `Microsoft.MixedReality.Toolkit.Providers.WindowsVoiceInput.WindowsDictationInputProvider.Update` :)

The script `CustomizeMrtk` is a (failed, for now) attempt to disable the dictation system automatically in such case (when `!PhraseRecognitionSystem.isSupported`). As a workaround, one can right now drag the `DictationOffProfile` in Unity over the appropriate (bottom-most) slot of the "Mixed Reality Toolkit" component in the "MixedRealityToolkit" GameObject.

If the new "HoloToolkit MRTK" would avoid these problems, then great, we can remove this attempt at that point:)